prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/tibco/TibjmsImpl.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.parodos.tasks.tibco;
+
+import com.tibco.tibjms.TibjmsConnectionFactory;
+
+import javax.jms.*;
```

Comment:
according to coding convention, `*` should be expanded

Reply:
what do u mean?

Reply:
I need many classes from that package. So I use '*'. That is super common.

Reply:
there are coding conventions to this project. we should respect them, else, each one will use its own style.

Reply:
conventions where? look at Java code all over the world. People prefer '*' then specifying a gazillion import lines. Besides, who says others are right. I will change it just for passing the PR. But this is no doubt the wrong thing to do.

Reply:
by the way, when you tell intellij to import a class then this is what intellij does. Do we know better? we know better than them what is the convention?